src/carts/pom.xml

```diff
@@ -15,7 +15,7 @@
     <properties>
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
-        <helidon.version>1.3.1</helidon.version>
+        <helidon.version>1.4.1</helidon.version>
         <jsonb.version>1.0.2</jsonb.version>
         <yasson.version>1.0.5</yasson.version>
```

Comment:
Why not bump version of yasson and jsonb?

Reply:
done